spec/models/post_spec.rb

```diff
@@ -1302,22 +1302,39 @@ def post_with_body(body, user = nil)
     end
   end
 
-  describe ".hide!" do
+  describe "#hide!" do
+    fab!(:post) { Fabricate(:post) }
+
     after do
       Discourse.redis.flushdb
     end
 
     it "should ignore the duplicate check" do
-      p1 = Fabricate(:post)
-      p2 = Fabricate(:post, user: p1.user)
+      post_2 = Fabricate(:post, user: post.user)
       SiteSetting.unique_posts_mins = 10
-      p1.store_unique_post_key
-      p2.reload.hide!(PostActionType.types[:off_topic])
-      expect(p2).to be_hidden
+      post.store_unique_post_key
+      post_2.reload.hide!(PostActionType.types[:off_topic])
+      expect(post_2).to be_hidden
+    end
```

Comment:
🤔 Sorry I don't have enough context here, but is there an assertion to be made for "ignoring the duplicate check"?

Reply:
I updated the existing spec to be clearer about what it is asserting for :)